frontend/src/app/finance/page.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Finance() {
+    return (
+      <main className="p-10">
```

Comment:
Make this a component:
example: <PageContainer></PageContainer>

export default const PageContainer () {
return (<main classname="p-10"></main>)
}